builds/e2e/nested-longhaul.yaml

```diff
@@ -1,119 +1,182 @@
 trigger: none
 pr: none
 
-variables:
-  images.artifact.name.linux: 'core-linux'
-  vsts.project: $(System.TeamProjectId)
-  # Variable defined in VSTS
-  #azure.keyVault: ''
-  #azure.subscription: ''
-  #edgelet.package.build: <edglet package build id>
-  #images.build: <images build id>
-  #pool.name: ''
-  #container.registry: ''
-  #snitch.storage.account: ''
-  # Variables settable at queue time
-  #test.startDelay.arm
-  #test.startDelay.amd
-  #edgelet.package.branchName
-  #images.branchName
-  #loadGen.message.frequency.amd64: 'xx:xx:xx.xx'
-  #loadGen.message.frequency.arm32v7: 'xx:xx:xx.xx'
-  #loadGen.message.frequency.arm64v8: 'xx:xx:xx.xx'
-  #run.linux.amd64.moby: 'true/false'
-  #run.linux.arm32v7.moby: 'true/false'
-  #run.linux.arm64v8.docker: 'true/false'
-  #longhaul.desiredModulesToRestartCSV: ''
-  #longhaul.restartIntervalInMins: ''
-  #twinTester.twinUpdateSize: ''
-  #twinTester.twinUpdateFrequency: 'xx:xx:xx.xx' 
-  #twinTester.twinUpdateFailureThreshold: 'xx:xx:xx.xx'
-  #twinTester.edgeHubRestartFailureTolerance: 'xx:xx:xx.xx'
-  #metricsCollector.metricsEndpointsCSV
-  #metricsCollector.scrapeFrequencyInSecs: 'xx:xx:xx.xx'
-  #metricsCollector.uploadTarget
+resources:
+  pipelines:
+  - pipeline: images
+    source: 'Azure-IoT-Edge-Core Build Images'
+    branch: master
+  - pipeline: packages
+    source: 'Azure-IoT-Edge-Core Edgelet Packages'
+    branch: master
 
-jobs:
-################################################################################
-  - job: linux_amd64_moby
-################################################################################
-    displayName: Linux AMD64 Moby
-    condition: ne(variables['agent.group'], '')
+stages:
+- stage: SetupVM
+  jobs:
+  - job: SetupVM_level5
+    displayName: SettingUp level 5
+    timeoutInMinutes: 180
+    variables:
+      artifactName: iotedged-ubuntu18.04-amd64
+      identityServiceArtifactName: packages_ubuntu-18.04_amd64
+      identityServicePackageFilter: aziot-identity-service_*_amd64.deb
+      vsts.project: $(System.TeamProjectId)
+      level: '5'
+      deploymentFile: 'nestededge_topLayerBaseDeployment.json'
+      parentName: ''
+      parentDeviceId: ''
+    pool:
+     name: $(pool.name)
+     demands:
```

Comment:
We need to put the nested edge long haul vm's in their own group so that they don't get used by another pipeline
We can do this by agent group number, but I feel it would be better to make a new capability and demand it here.
For example: new capability on the agents "nested-long-haul" with value "true"
And then demand that nested-long-haul is true here

Reply:
That's a good point but `agent.group` is from pipeline ENV variables. We could create new `agent.group` for long haul tests.

Reply:
We can, but it'll be confusing
a) we have to keep track of which number and make sure other pipelines don't cross to it
b) If we have multiple nested long haul pipelines in the future (we have multiple long haul pipelines right now), it gets even more confusing